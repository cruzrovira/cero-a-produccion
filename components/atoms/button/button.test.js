import render from "react-test-renderer"

import Button from "./button"
describe("test de botton", () => {
  it("Button Default", () => {
    const button = render.create(<Button> Button Default </Button>)
    expect(button.toJSON()).toMatchSnapshot()
  })

  it("Button Primary ", () => {
    const button = render.create(
      <Button type="primary"> Button primary </Button>
    )
    expect(button.toJSON()).toMatchSnapshot()
  })

  it("Button Secondary ", () => {
    const button = render.create(
      <Button type="secondary"> Button Secondary </Button>
    )
    expect(button.toJSON()).toMatchSnapshot()
  })

  it("Button Tertiary ", () => {
    const button = render.create(
      <Button type="tertiary"> Button Tertiary </Button>
    )
    expect(button.toJSON()).toMatchSnapshot()
  })
})
