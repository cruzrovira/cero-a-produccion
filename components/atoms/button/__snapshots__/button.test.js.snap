// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test de botton Button Default 1`] = `
<button
  className="button"
>
   Button Default 
</button>
`;

exports[`test de botton Button Primary  1`] = `
<button
  className="button type-primary"
>
   Button primary 
</button>
`;

exports[`test de botton Button Secondary  1`] = `
<button
  className="button type-secondary"
>
   Button Secondary 
</button>
`;

exports[`test de botton Button Tertiary  1`] = `
<button
  className="button type-tertiary"
>
   Button Tertiary 
</button>
`;
